sycl/doc/GettingStartedAIE.md

```diff
@@ -0,0 +1,325 @@
+export AIE_ROOT="$XILINX_ROOT/Vitis/$VITIS_VERSION/$AIE_NAME"
+export XILINX_VITIS_AIETOOLS=$AIE_ROOT
+export PATH="$PATH:$AIE_ROOT/bin/unwrapped/lnx64.o/"
+source $AIE_ROOT/scripts/cardano_env.sh
+export CHESSROOT=$AIE_ROOT/tps/lnx64/
+```
+
+## Several flavors for compiling a basic AIE++ Hello World
+
+So for these examples we will compile the [hello_world.cpp](sycl/test/acap_tests/hello_world.cpp) 
+test. It's a 1 processing element (AI Engine core) kernel that prints some 
+output.
+
+In the following examples we assume the environment variable *SYCL_BIN_DIR* is
+setup to point the build/bin directory of the compiled SYCL device compiler. 
+Although in the two cases where we're only compiling for CPU it's not strictly 
+required, provided the C++ compiler you use is fairly up-to-date and supports 
+C++ 20, the commands should work.
+
+The directory `/net/xsjsycl41/srv/Ubuntu-20.10/arm64-root-server-rw-tmp` we 
+make reference to in the cross-compilation commands is an Ubuntu-19.04 arm64 
+(AKA aarch64) image setup to contain all of the required libraries we need for 
+cross-compilation.
+
+In all of the following commands we assume you're running the latest
+version of Ubuntu Debian.
+
+### Compiling a basic AIE Hello World for native CPU execution
+
+The following compilation command will compile the hello_world.cpp example for 
+your native CPU. What is meant by native in this case, is whatever architecture 
+you're compiling on. So compiling on x86, will result in a hello_world 
+binary that executes on an x86 CPU. All kernels will also execute on the CPU,
+no device offloading or compilation is done.
+
+```bash
+  $SYCL_BIN_DIR/clang++ -std=c++20 hello_world.cpp -I/AIE++/acappp/include \
+    `pkg-config gtkmm-3.0 --cflags` `pkg-config gtkmm-3.0 --libs`
+```
+### Cross-compilation for ARM CPU execution
+
+The following compilation command will compile the hello_world.cpp example for 
+ARM CPU. The general idea of the compilation commands is that you're compiling 
+your binary on an Linux x86 host for an Linux ARM64 (aarch64) target.
+ 
+In this case the assumption is that the ARM target is the AIE PS which is an 
+Cortex-A72 CPU, but in theory it could be whatever `-mcpu` target the Clang 
+compiler supports. 
+ 
+```bash
+  $ISYCL_BIN_DIR/clang++ -std=c++20 -target aarch64-linux-gnu -mcpu=cortex-a72 \
+    --sysroot /net/xsjsycl41/srv/Ubuntu-19.04/arm64-root-server-rw-tmp \
+    `pkg-config gtkmm-3.0 --cflags` `pkg-config gtkmm-3.0 --libs` \
+    -I/AIE++/acappp/include  hello_world.cpp
+```
+
+### Cross-compilation for AIE PS/AI Engine execution
+
+This is the only compilation command that requires the SYCL device compiler. 
+As the intent is to compile all of the kernels for the device to be offloaded, 
+and the rest for the host.
+
+From a SYCL perspective the AIE PS is the host device that will offload to
+the other elements of the AIE architecture. The device in this case is the AIE 
+AI Engine cores.
```

Comment:
Technically the AIE has no PS but has compute tiles, shim tiles and mem tiles after AIE1.
The PS is outside of AIE, such as ACAP.